www/includes/easyparliament/templates/html/people/mp_index.php

```diff
@@ -0,0 +1,226 @@
+                        <a href="<?= $urls['plain'] ?>?date=1970-06-18">MPs at 1970 general election</a>
+                        <a href="<?= $urls['plain'] ?>?date=1966-03-31">MPs at 1966 general election</a>
+                        <a href="<?= $urls['plain'] ?>?date=1964-10-15">MPs at 1964 general election</a>
+                        <a href="<?= $urls['plain'] ?>?date=1959-10-08">MPs at 1959 general election</a>
+                        <a href="<?= $urls['plain'] ?>?date=1955-05-26">MPs at 1955 general election</a>
+                        <a href="<?= $urls['plain'] ?>?date=1951-10-25">MPs at 1951 general election</a>
+                        <a href="<?= $urls['plain'] ?>?date=1950-02-23">MPs at 1950 general election</a>
+                        <a href="<?= $urls['plain'] ?>?date=1945-07-05">MPs at 1945 general election</a>
+                        <a href="<?= $urls['plain'] ?>?date=1935-11-14">MPs at 1935 general election</a>
+                        <label for="past-list-custom-date">Custom date&hellip;</label>
+                        <span class="input-appended">
+                            <input type="text" id="past-list-custom-date" name="date" class="form-control" placeholder="YYYY-MM-DD">
+                            <input type="submit" value="Download" class="button">
+                        </span>
+                    </p>
+                </form>
+
+                <script type="text/javascript">
+                $(function(){
+                  $('#past-list-dates').hide();
+                  $('a[href="#past-list-dates"]').on('click', function(e){
+                    e.preventDefault();
+                    $(this).trigger('blur');
+                    $('#past-list-dates').slideToggle();
+                  })
+                });
+                </script>
+                <?php } else if ( $type == 'msps' ) { ?>
+                    <p class="past-list-dates" id="past-list-dates">
+                        <a href="<?= $urls['plain'] ?>?date=2007-05-03">MSPs at 2007 general election</a>
+                        <a href="<?= $urls['plain'] ?>?date=2003-05-01">MSPs at 2003 general election</a>
+                        <a href="<?= $urls['plain'] ?>?date=1999-05-06">MSPs at 1999 general election</a>
+                        <a href="<?= $urls['plain'] ?>?all=1">Historical list of all MSPs</a>
+                    </p>
+                <?php } else { ?>
+                    <p class="past-list-dates" id="past-list-dates">
+                    <a href="<?= $urls['plain'] ?>?all=1">Historical list of all <?= $rep_plural ?></a>
+                    </p>
+                <?php } ?>
+
+                <?php include( dirname(__FILE__) . '/../sidebar/looking_for.php' ) ?>
+            </div>
+        </div>
+
+    </div>
+</div>
+
+<script type="text/javascript">
+$(function(){
+  $('#download-past-list').on('change', function(){
```

Comment:
There doesn't appear to be this ID anywhere, not sure what this JS section is for.